quickfixj-core/src/main/java/quickfix/mina/initiator/AbstractSocketInitiator.java

```diff
@@ -123,6 +123,11 @@ private void createInitiator(final Session session, final boolean continueInitOn
             throw new ConfigError("Must specify at least one socket address");
         }
 
+        int connectTimeout = 60; // 1 minute by default, matches MINA
+        if (getSettings().isSetting(sessionID, Initiator.SETTING_SOCKET_CONNECT_TIMEOUT)) {
+            connectTimeout = getSettings().getInt(sessionID, Initiator.SETTING_SOCKET_CONNECT_TIMEOUT);
+        }
```

Comment:
This could be simplified using SessionSettings.getIntOrDefault(60)


```suggestion
        // 1 minute by default, matches MINA
        int connectTimeout = getSettings().getIntOrDefault(sessionID, Initiator.SETTING_SOCKET_CONNECT_TIMEOUT, 60);
        }
```